docassemble/AssemblyLine/al_document.py

```diff
@@ -709,8 +712,8 @@ def __getitem__(self, key):
     # point to the same file.
     return self
   
-  def as_list(self, key:str='final', refresh:bool=True) -> List[DAFile]:
-    return [self]
+  def as_list(self, key:str='final', refresh:bool=True) -> List[DAStaticFile]:
+    return [self[key]]
```

Comment:
This one is just for typing, yeah. But mypy isn't quite happy with it, since technically `DAStaticFile` and `DAFile` are different types (probably to simplify the implementation of DAStaticFile). 

Either way, yeah, this one can be removed.

Reply:
```suggestion
    return [self]
```